st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -88,7 +91,19 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        result = {
+            'execution': execution,
+            'actionalias': ActionAliasAPI.from_model(action_alias_db)
+        }
+
+        if action_alias_db.ack and 'format' in action_alias_db.ack:
+            jinja = jinja2.Environment(trim_blocks=True, lstrip_blocks=True)
```

Comment:
Any reason this 2 lines of jinja config are applicable only in this context i.e. any reason other place where we use jinja this may not be required? 

Please see if https://github.com/StackStorm/st2/blob/master/st2common/st2common/util/jinja.py#L127 can be used here so we have a consistent jinja experience though stackstorm.

also, check if those 2 lines should be included.